etn-ccis/blui-react-workflows
# Add blui-ci github action

Fixes BLUI-5563.



#### Changes proposed in this Pull Request:

- add blui-ci github workflow
- remove circleci config and stopped build in circleci dashboard
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- to view workflow jobs and dist artifact https://github.com/etn-ccis/blui-react-workflows/actions/runs/8469705988
- make small commit to readme to see workflow run and revert commit
- codecov upload from test job https://app.codecov.io/gh/etn-ccis/blui-react-workflows/tree/feature%2Fblui-5563-blui-ci



#### Any specific feedback you are looking for?

-
